.github/CODEOWNERS

```diff
@@ -7,4 +7,4 @@ codemodernizer/ @aws/elastic-gumby
 codetransform/ @aws/elastic-gumby
 codewhisperer/ @aws/codewhisperer-team
 
-mynah-ui/ @aws/dexp
```

Comment:
i think we wanted this reviewer change to be specific to the NPM package updates instead of everything in here as it'll cause friction with partners